c/meterpreter/source/extensions/python/Modules/signalmodule.c

```diff
@@ -7,7 +7,7 @@
 #include "intrcheck.h"
 
 #ifdef MS_WINDOWS
-#include <windows.h>
+#include <Windows.h>
```

Comment:
you might still need this commit

Reply:
though now I see it's in the python extension, which probably isn't the best thing to target for the first pass

Reply:
You're right, this commit is needed.

This won't break the build with Visual Studio because it manages includes in a case-insensitive way, as opposed to MinGW.